.github/workflows/unit-test.yml

```diff
@@ -152,8 +172,11 @@ jobs:
               with:
                   custom-cache-suffix: $(/bin/date -u --date='last Mon' "+%F")
 
-            - name: Npm build
-              run: npm run build
+            - name: Download assets
+              uses: actions/download-artifact@c850b930e6ba138125429b7e5c93fc707a7f8427 # v4.1.4
+              with:
+                  name: build-assets
+                  path: ./build
 
             - name: Docker debug information
               run: |
```

Comment:
Why do we have docker debug separate from general debug? Shall we combine this step and the next one?

Reply:
This logs the version of Docker installed within the runner image. The other debug steps below this one log the running containers, and then some debug detail about what's running within the containers.

I think it makes sense to keep them separate becuase they have different contexts, but don't feel strongly here.

Reply:
My question was about the following "General debug information" immediately following, not about the other docker logging like the running containers. It seems like the docker version could be printed there and we could save a step, it's mostly printing versions and it doesn't seem like there's any reason to isolate the docker version.

It's not too important and can be done in another PR if we'd like.